### ISSUE TITLE
Show plugins README inline

### ISSUE THREAD

**Issue by Globegitter** [getgrav/grav-plugin-admin#1114]
I am just testing out grav for the first time to consider using for our company website (instead of wordpress). First of all, great job so far love the directions this is heading - making it a nice experience for developers as well as non-tech content authors.

I think it would be nice if the README file on the plugin info page (so e.g. `admin/plugins/star-ratings`) could be displayed inline. Would save a bit of time but could also make it a bit more nicer for the more non-tech people using grav and not necessarily knowing what github is, maybe being a bit more reluctant to click on that link (I am imagining this to be one of the comments if we where to start using grav right now from some of our team-members)

**Comment by OleVik** [comment]
Since the README is self-contained in the plugin-folder, it could just be pulled into a modal and have its Markdown parsed. Shouldn't take all that much to implement, but opening the README in a modal versus opening it on GitHub does not necessarily improve the situation for the end-user - the README might not be written any more clearly than settings in blueprints are described.

GitHub is obviously a safe place, and renders READMEs nicely, but I think the focus needs to be on better specifications and explanations in blueprints - and all uses of the plugin should be apparent from the blueprint, inasmuch as the plugin integrates with Admin.

**Comment by Globegitter** [comment]
Yep of course I know GitHub is a safe place, but outside the developer world not everyone knows that. But you are making a good point, maybe the README itself is best kept on GitHub. I am not sure though if a yaml file is the best place to have an extended description of something. And most plugins that I have looked at have very short descriptions, making it quite difficult to judge what I am actually installing, just while staying in the admin ui.

E.g. for the star ratings plugin that is what I see:

![image](https://cloud.githubusercontent.com/assets/2580254/25964378/ece59ba2-3684-11e7-9f3b-b9190a51ad0a.png)

Where-as on the README I get much more information (though some information not relevant to the admin ui): https://github.com/getgrav/grav-plugin-star-ratings/blob/develop/README.md. Reading the `Description` as well as part of the `Configuration` in that file I actually feel I know what I am installing.

I think it would just be nice to have some way of getting better descriptions, maybe even including pictures if relevant, all within the grav gui.

**Comment by karfau** [comment]
I totlly agree with the argumentation. the list of plugins is getting longer and deciding which one to use would get much easier based on some relevant stats from github about a plugin: like stars, last commit/release, number of contributors/forks, maybe badges...

As this information is publicly available it could just be fetched using javascript, and added if available.

Do you guys think this should be part of this plugin or would you prefer to have a separate plugin that extends the admin plugin in this way?
